src/main/java/org/apache/maven/shared/filtering/DefaultMavenResourcesFiltering.java

```diff
@@ -254,9 +255,15 @@ private File getTargetFile(File file) throws MavenFilteringException {
                 Path destination = getDestinationFile(outputDirectory, targetPath, "", mavenResourcesExecution)
                         .getAbsoluteFile()
                         .toPath();
+                String origin = basedir.relativize(
+                                resourceDirectory.getAbsoluteFile().toPath())
+                        .toString();
+                if (StringUtils.isEmpty(origin)) {
+                    origin = "base directory";
```

Comment:
classical value is `.`, isn't it?

Reply:
Are you suggesting to put "." in the message?

Reply:
yes, `.` instead of `base directory`

Reply:
Done, thanks!